### DISCUSSION TITLE
Feature request - Add backup options under account page

### DISCUSSION THREAD

**Discussion by techrivertree** [mealie-recipes/mealie#3554] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I lost my entire Mealie database when I forgot the login info for my account. I reinstalled mealie and I really like the updated interface but there is no option to backup your configuration from the account page. Reading the docs, I am told to "Navigate to /admin/backups". Since I have no idea how to do that, can a backup option be added under the account page?

### Please provide a concise description of the feature that would resolve your issue.

Add backup option under account page

### Please consider and list out some caveats or tradeoffs made in your design decision

Hopefully this will save other people from rebuilding their databases like I had to do.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [X] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
> I lost my entire Mealie database when I forgot the login info for my account

You can reset your password from the command line:

https://docs.mealie.io/documentation/getting-started/faq/#how-can-i-change-my-password

> Navigate to /admin/backups

You can go directly to the URL: `https://demo.mealie.io/admin/backups`, or you click on "Settings" and then "Backups":

![image](https://github.com/mealie-recipes/mealie/assets/71845777/c0075490-0f2d-4e12-b1f4-5e4f3140e155)

![image](https://github.com/mealie-recipes/mealie/assets/71845777/97d611dd-e8a7-4540-8111-73307ce479c6)

**Comment by techrivertree** [comment]
There is no "settings " menu on my mealie install.

![Screenshot from 2024-05-03 13-27-12](https://github.com/mealie-recipes/mealie/assets/69227859/1109198b-1266-4800-ab12-02930c8326b1)

**Reply by michael-genson** [reply]
Only admins can access settings. Are you an admin? You can check at `/group/members` (demo: https://demo.mealie.io/group/members):

![image](https://github.com/mealie-recipes/mealie/assets/71845777/e02554e8-4796-4114-9e89-303929d69303)

**Comment by techrivertree** [comment]
I see that I am only a user. I have no idea how to change my account type.

**Comment by michael-genson** [comment]
If you're the only user on the instance, and you're not an admin, you likely deleted the only admin user on the instance. As far as I know there's no easy way to fix this, you need to manually edit the database and make yourself an admin.

Either that, or there is another user in another group with admin privileges (which wouldn't appear in the user list), and only that user can make you an admin (without directly editing the database).

**Reply by techrivertree** [reply]
I guess I will be reinstalling Mealie. FYI, Mealie is installed on TrueNAS scale.
